### PR TITLE
Added the nav link of contact section to the nav bar and updated the behavior of nav links upon scrolling over sections

### PR DESCRIPTION
**Pull request by K-kiran77**
The nav link to the contacts section is added and previous when i clicked on any of the nav link the nav link remains active even if i scrolled to another section, i have added event listeners to each of the section (home, about, blogs, resource library and contact), what happens by doing this is when you click on any of the nav link the nav link is active only until and unless scrolled to another section,  when one nav link is active and if we scroll to another section the nav link corresponding to the current section become active and the previously activated nav link gets deactivated resulting in better user experience

Thank You @Suchitra-Sahoo for assigning this project to me under GSSoC 2024

It was such an amazing experience for me adding some features to this project

Once again thank you @Suchitra-Sahoo

### CODE REVIEW

**Comment by Suchitra-Sahoo** [comment]
Mention your issue number.

**Comment by Suchitra-Sahoo** [comment]
Add some screenshot or video to see your changes.

**Comment by K-kiran77** [comment]
> Mention your issue number.

My issue number is #80


> Add some screenshot or video to see your changes.

before

https://github.com/Suchitra-Sahoo/AgriLearnNetwork/assets/132844726/8fc62a4d-e6e3-419a-8996-5d4fb9182b77

after

https://github.com/Suchitra-Sahoo/AgriLearnNetwork/assets/132844726/140fe27c-756e-4b47-b0a7-0ffa9aec7c04

File: index.html [hunk 1]
```diff
@@ -32,16 +32,17 @@
       <a href="#about"><i class="fa-solid fa-circle-info"></i>  About</a>
       <a href="#Farm Management"> <i class="fa-solid fa-blog"></i>  Blogs</a>
       <a href="#Resource Library"><i class="fa-solid fa-book"></i>  Resource Library</a>
+      <a href="#contact" id="nav--contact"><i class="fa-solid fa-phone"></i> Contact</a>
```
Comment (K-kiran77) [inline index.html:36]: Added new nav link for the contact section in the navbar

**Comment by Suchitra-Sahoo** [comment]
Even though I have merged your PR but when I scroll down I can see the navbar options are not activating accordingly. Look into it.

**Comment by K-kiran77** [comment]
> Even though I have merged your PR but when I scroll down I can see the navbar options are not activating accordingly. Look into it.

I have found the issue someone has changed the css that's why it is not working, The css code in machine is when i send pull request is different from the present css file i you allow me i will change the css file and bring back the feature

**Comment by Suchitra-Sahoo** [comment]
This might have happened because recently various PR has been merged. You may pull all changes from origin and raise another PR.

**Comment by K-kiran77** [comment]
The website is working fine now with issue I raised, I think again after some recent PRs it starts working in the expected way

**Comment by K-kiran77** [comment]
could you please give me the level label  to my issue under GSSoC, previously you have given me only the gssoc label but not the type of level of my issue #80 

Thank You @Suchitra-Sahoo

**Pull request merged**